bin/fail2ban-client

```diff
@@ -144,32 +144,48 @@ class Fail2banClient:
 		return self.__processCmd([["ping"]], False)
 
 	def __processCmd(self, cmd, showRet = True):
-		beautifier = Beautifier()
-		streamRet = True
-		for c in cmd:
-			beautifier.setInputCmd(c)
-			try:
-				client = CSocket(self.__conf["socket"])
-				ret = client.send(c)
-				if ret[0] == 0:
-					logSys.debug("OK : " + `ret[1]`)
+		streamRet = False
+		if self.__hasSocketAccess(showRet):
+			beautifier = Beautifier()
```

Comment:
You're right, for the wrong reason (imho) :)   I made my change thinking __processCmd "processed a command"...negating the loop of potentially multiple commands...any of which could fail or succeed independently and during which the socket could vanish